### DISCUSSION TITLE
Where is the author's tag?

### DISCUSSION THREAD

**Discussion by yarcowong** [todotxt/todo.txt#81] [Ideas]
When a team is doing a project, normally we are going to add

```
// TODO: something need to be done in future
```

But if there's another guy are writting todos, things are mixed together. So, I'd prefer to add author:

```
// TODO (yarco): something need to be done in future
```

**Comment by bobwhitelock** [comment]
Todo.txt doesn't define or restrict the extra tags you add to  a todo, so you can do this by adding your own `author` tag to each one:

```
// TODO: something need to be done in future author:yarco
```
